src/plugins/core/chart.ts

```diff
@@ -100,6 +100,7 @@ export class ChartPlugin extends CorePlugin<ChartState> implements ChartState {
                 size: { width: fig.width, height: fig.height },
                 definition: chart.getDefinition(),
                 sheetId: cmd.sheetIdTo,
+                sheetMap: this.getters.getSheetMap(),
```

Comment:
We talked about filtering the map with only relevant sheet name to reduce the size. Is it annoying/not necessary ?